audio_effects.conf

```diff
@@ -204,14 +213,14 @@ effects {
     library loudness_enhancer
     uuid fa415329-2034-4bea-b5dc-5b381c8d1e2c
   }
-  aec {
-    library qcom_pre_processing
-    uuid 1e5c3ea0-1fcf-11e3-9176-0002a5d5c51b
-  }
-  ns {
-    library qcom_pre_processing
-    uuid 25ec95c0-1fcf-11e3-b5f8-0002a5d5c51b
-  }
+#  aec {
```

Comment:
Do not comment out stuff. If its not needed, delete it. Commented code and configs blow up the files and make it harder to see exactly whats going on.